modules/ROOT/pages/index.adoc

```diff
@@ -2,15 +2,19 @@
 
 [IMPORTANT]
 ====
-OpenShift Serverless - Developer Preview releases contain features and functionalities that might not be fully tested. Customers are encouraged to provide feedback on Developer Preview releases.
+OpenShift Serverless - Technology and Developer Preview releases contain features and functionalities that might not be fully tested. Customers are encouraged to provide feedback on Technology and Developer Preview releases.
 
-Developer Preview releases are not production-ready, and customers are recommended to avoid using the project for production or business-critical workloads.
+Technology and Developer Preview releases are not production-ready, and customers are recommended to avoid using the project for production or business-critical workloads.
+
+OpenShift Serverless provides information about additional, advanced use cases and integrations that are possible using OpenShift Container Platform and OpenShift Serverless, but are for technology and development purposes and are not officially supported by Red Hat.
+
+OpenShift Serverless is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.

```

Comment:
1. @kaldesai please remove the paragraph from line 11. The rest of the content looks good from the over view pov.
2. In https://deploy-preview-99--jazzy-shortbread-5f62b7.netlify.app/docs/latest/serverless-eventing/service-mesh/eventing-service-mesh-setup, and https://deploy-preview-99--jazzy-shortbread-5f62b7.netlify.app/docs/latest/functions/serverless-functions-about sections, please add notes which provide Developer Preview related disclaimer:
OpenShift Serverless - Developer Preview releases contain features and functionalities that might not be fully tested. Customers are encouraged to provide feedback on Developer Preview releases.
Developer Preview releases are not production-ready, and customers are recommended to avoid using the project for production or business-critical workloads.
3. In https://deploy-preview-99--jazzy-shortbread-5f62b7.netlify.app/docs/latest/serverless-logic/about please add the Technology preview related note:
'Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production.
These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.'
Basically, the overview conveys that this site covers info that is DP and TP  and the respective section mention that they are TP or DP respectively.

Reply:
I have incorporated all of your changes.